shared/keys/main.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
@Jasonyou1995 When we started working on #320, we ran into an issue where all methods in a directory and its subdirectories were different packages, because every directory in a Go package. Ref: https://www.codementor.io/@inanc/understanding-go-packages-cgsx1hzrx

This is a separate project (not a separate package), which needs a new main file, go.sum, go.mod, and there should only be one in any project. For example, VRF (https://github.com/pokt-network/pocket/tree/main/consensus/leader_election/vrf) is a package in the same project that is not integrated into the end-to-end flow.

Lmk if that makes sense. I can review the core work once this change is made.

Reply:
Yes! I will update the code for a better re-usability and merge all code from key-base into the pocket project. Thanks for pointing out and sharing the article!